Lib/glyphsLib/classes.py

```diff
@@ -2213,6 +2262,15 @@ def bounds(self):
     #     lambda self: self.piece,
     #     lambda self, value: setattr(self, "piece", value))
 
+    def draw(self, pen: AbstractPen) -> None:
+        """Draws component with given pen."""
+        pointPen = PointToSegmentPen(pen)
+        self.drawPoints(pointPen)
```

Comment:
segment pens also have a similar `addComponent` method, you can call that directly here I guess